org.mixedrealitytoolkit.uxcomponents/Experimental/NonNativeKeyboard/TouchableNonNativeKeyboard.prefab

```diff
@@ -0,0 +1,174 @@
+%YAML 1.1
```

Comment:
Please add change comment to "UX Component's" CHANGELOG.md.   Following the [Keep A Changelog](https://keepachangelog.com/en/1.1.0/) pattern.

Also please change that "UX Component's version has been properly incremented since last [release](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/releases).

Also please make sure "UX Component's" dependency versions have been properly incremented. Since you are depending on new feature in "UX Core"

Reply:
I have updated the ChangeLog but I have no idea what you mean by "Also please change ... " What am I supposed to do? Edit the asmdef?

Reply:
I believe this would be updating the package.json to "version": "3.2.0-development". Last version of UX Core was 3.1.1

Reply:
The dependencies in the package.json file will also need to be updated, since this depends on the spatial manipulation and uxcore changes you are making

Reply:
I think I fixed what you asked. I hope I did it correctly